src/Sarif.Converters/TSLintConverter.cs

```diff
@@ -97,10 +97,10 @@ internal Result CreateResult(TSLintLogEntry entry)
 
             Uri analysisTargetUri = new Uri(entry.Name, UriKind.Relative);
```

Comment:
Let's rename `analysisTargetUri` to `uri`. We're now reserving "analysis target" to mean "the thing I was asked to scan that led me to discover a result in some other file."